### PR TITLE
DSPDC-1025 Split out separate template for file lookup.

### PR DESCRIPTION
**Pull request by danxmoran**
(We'll still need to look up the file in ClinVar before we attempt to ingest, but the behavior for what-to-do when a file is already present will be more complex than we have here now)

### CODE REVIEW

File: charts/argo-templates/scripts/lookup-file-id.py [hunk 1]
```diff
@@ -11,14 +11,16 @@
...
+def get_file_id(target_path: str):
     response = authed_session.get(f"{base_url}/api/repository/v1/datasets/{dataset_id}/filesystem/objects",
                                   params={"path": target_path})
     if response.status_code == 200:
-        return "true"
-    elif response.status_code == 404:
-        return "false"
-    else:
+        return response.json()["fileId"]
+    elif response.status_code != 404:
```
Comment (raaidbroad) [inline charts/argo-templates/scripts/lookup-file-id.py:19]: what happens if it is a 404?

Comment (danxmoran) [reply charts/argo-templates/scripts/lookup-file-id.py:19]: (I think) the function will "return" `None`

Comment (raaidbroad) [reply charts/argo-templates/scripts/lookup-file-id.py:19]: this is a pythonism that I don't like haha, apparently you can just leave off an else. 🤷 fine by me then

**Review by raaidbroad — APPROVED** [review]

File: charts/argo-templates/templates/ingest-file.yaml [hunk 2]
```diff
@@ -19,26 +19,10 @@ spec:
           - name: sa-secret-key
       dag:
         tasks:
-          # 1) Make sure that something doesn't already exist at the target path
```
Comment (quazi-h) [inline charts/argo-templates/templates/ingest-file.yaml:22]: Glad we removed these numbered steps - hopefully will be less confusing to extend/change things in the future.

**Review by quazi-h — APPROVED** [review]

**Pull request merged**